### ISSUE TITLE
Help please

### ISSUE THREAD

**Issue by jrhbright** [Burningstone91/Hue_Dimmer_Deconz#1]
This is exactly what I am looking for, but I'm not sure how to get it to work.   (I'm a relative newbie!)

I've installed it (successfully?) using HACS.  But I'm not sure what to do next. Where does the config go?

Got the hue and deCONZ working well, with a dimmer switch ready to go in doCONZ.

Sorry to be a pai

**Comment by Burningstone91** [comment]
Are you running Hass.io? Is AppDaemon up and running?
If you installed the app through HACS, it should be stored under this directory `/config/appdaemon/apps/hue_dimmer_switch_deconz`
You either addthe configuration to the file apps.yaml in the directory
`/config/appdaemon/apps` or add a new YAML file in the folder and add the config there if you want to separate the files.

**Comment by jrhbright** [comment]
Thanks so much, yes using hass.io, I was just researching how to write scripts and automations to handle each of the button presses when I found this, but it is the first time I've used AppDaemon so a bit of a steep learning curve. I've got it working well now thanks to your advice I was missing a couple of things.  

For the benefit of anyone else as new to this as me:
- I installed AppDaemon via the hass.io addin.  Updated the configuration in `/config/appdaemon/appdaemon.yaml` according to instructions here: [https://appdaemon.readthedocs.io/en/latest/CONFIGURE.html#initial-setup](url)
- I installed HACS using the guide here: [https://hacs.xyz/docs/installation/manual](url)
- I installed Hue_Dimmer_Deconz via HACS and have found the directory `/config/appdaemon/apps/hue_dimmer_switch_deconz` all present and correct.
- I added an `apps.yaml` file in the directory `/config/appdaemon/apps` alongside the `hue_dimmer_switch_deconz` directory.
- My apps.yaml file looks like this:
```
    dimmer_living_room:
      module: hue_dimmer_switch_deconz
      class: HueDimmerSwitch
      entities:
        switch: living_room_dimmer
        light: light.living_room_main_light
```
Boom, thanks so much.
